scripts/copy_crds.sh

```diff
@@ -10,5 +10,5 @@ CRD_PATH="${SCRIPT_ROOT}/vendor/github.com/operator-framework/api/crds"
 rm "${SCRIPT_ROOT}"/deploy/chart/crds/*.yaml
 for f in "${CRD_PATH}"/*.yaml ; do
     echo "copying ${f}"
-    cp "${f}" "${SCRIPT_ROOT}/deploy/chart/crds/0000_50_olm_00-$(basename "$f" | sed 's/^.*_\([^.]\+\)\.yaml/\1.crd.yaml/')"
+    cp "${f}" "${SCRIPT_ROOT}/deploy/chart/crds/0000_50_olm_00-$(basename "$f" | sed 's/^.*_\([^.]\{1,\}\)\.yaml/\1.crd.yaml/')"
```

Comment:
why is this needed?

Reply:
This is needed to make the code portable so it can be run on linux/Mac versions of sed. Here's what I wrote in the commit comment:

```
The command + in sed is non-portable. The posix compliant version of
this command is {1,}

This change allows this script to run on a mac too (which uses BSD sed)

See https://riptutorial.com/sed/topic/9436/bsd-macos-sed-vs--gnu-sed-vs--the-posix-sed-specification
```

Reply:
Sorry.. git comment is slightly misleading since its missing some of the slashes:

The command `\+` in sed is non-portable. The posix compliant version of
this command is `\{1,\}`

This change allows this script to run on a mac too (which uses BSD sed)

See https://riptutorial.com/sed/topic/9436/bsd-macos-sed-vs--gnu-sed-vs--the-posix-sed-specification